rcldotnet/RCLdotnet.cs

```diff
@@ -199,8 +199,14 @@ public class RCLdotnet {
     public static Node CreateNode (string nodeName, string nodeNamespace) {
       IntPtr nodeHandle = IntPtr.Zero;
       int ret = RCLdotnetDelegates.native_rcl_create_node_handle (ref nodeHandle, nodeName, nodeNamespace);
-      Node node = new Node (nodeHandle);
-      return node;
+
+      if (ret != -1)
+      {
+        Node node = new Node (nodeHandle);
+        return node;
+      } else {
```

Comment:
If no node is created, this should raise an exception.

Reply:
Ok. I added a file for exceptions, with a new custom exception `InvalidNodeNameException`. It is raised when the node name is not correct.

Reply:
There's a list of possible return values in `https://github.com/esteve/ros2_dotnet/blob/master/rcldotnet_common/RCLRet.cs`, you can use it to convert from the native result to a `RCLRet` object.

Reply:
I didn't realize it. Fixed using these return types. I also added a `rcldotnet_ret.h` to be coherent with this in the native code.